docs/usage.md

```diff
@@ -0,0 +1,94 @@
+# Arcadia-Science/metagenomics Usage
+
+This page provides documentation on how to use the Arcadia-Science/metagenomics pipeline.
+
+## Input specifications
+
+This pipeline processes metagenomic reads in FASTQ format from either Illumina or Nanopore technologies. **Important note**: This pipeline separately processes Illumina or Nanopore metagenomes and does not perform hybrid assembly, polishing of Nanopore assemblies with Illumina reads, or scaffolding with Nanopore reads.
+
+For Nanopore reads, the pipeline takes as direct input a single fastq file and does not perform demultiplexing or basecalling, so it expects that those steps should be done prior to feeding into the pipeline and aggregating into a single fastq file.
+
+The workflow takes a CSV samplesheet listing the sample names and paths to the fastq files as direct input. It does not take the path to the directory of the fastq files, you must list them in a samplesheet. The CSV must contain the columns:
+
+`sample, fastq_1, fastq_2`
+Neither the `sample` or `fastq*` columns may contain spaces, and the `fastq` columns contains the path(s) to your fastq file(s).
+
+If you are inputting Illumina reads, the samplesheet looks like the following:
+
+```
+sample,fastq_1,fastq_2
+comm_1,https://github.com/Arcadia-Science/test-datasets/raw/main/metagenomics/illumina/comm_1_subsampled_1.fq.gz,https://github.com/Arcadia-Science/test-datasets/raw/main/metagenomics/illumina/comm_1_subsampled_2.fq.gz
+vir_1,https://github.com/Arcadia-Science/test-datasets/raw/main/metagenomics/illumina/vir_1_subsampled_1.fq.gz,https://github.com/Arcadia-Science/test-datasets/raw/main/metagenomics/illumina/vir_1_subsampled_2.fq.gz
+```
+
+If you are inputting Nanopore reads, the samplesheet looks like the following:
+
+```
+sample,fastq_1,fastq_2
+om,https://github.com/Arcadia-Science/test-datasets/raw/main/metagenomics/ont/om_subset_reads.fq.gz,
+el,https://github.com/Arcadia-Science/test-datasets/raw/main/metagenomics/ont/el_subset_reads.fq.gz,
+```
+
+Note that even for Nanopore reads which the input is in a single fastq file, still include the third column for `fastq_2` - the pipeline for processing Nanopore reads will ignore this and process your Nanopore reads that are in a single fastq file.
+
+## Running the pipeline
+
+A typical command for running the pipeline looks like:
+` nextflow run Arcadia-Science/metagenomics --input <SAMPLESHEET.csv> --outdir <OUTDIR> --platform <illumina|nanopore> -profile docker`
```

Comment:
I like this example of platform more than what you use in the main readme!

Reply:
+1

Reply:
also WAIT! if I have nextflow installed, will this just work without cloning the repo??? if that works that is very cool. can you specify which version or commit or release to pull?

Reply:
I thought with the `-r` flag you could specify the release to launch from without cloning the repo